packages/crowbar/crowbar.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "stephen.dolan@cl.cam.ac.uk"
+authors: ["Stephen Dolan"]
+homepage: "https://github.com/stedolan/crowbar"
+bug-reports: "https://github.com/stedolan/crowbar/issues"
+dev-repo: "https://github.com/stedolan/crowbar.git"
+license: "MIT"
+build:
+[[ "jbuilder" "build" "--only-packages" name "--root" "." "-j" jobs "@install" ]]
+build-test:
+[[ "jbuilder" "build" "--only-packages" name "--root" "." "-j" jobs "@runtest" ]]
+available: [ ocaml-version >= "4.02" ]
+depends: [
+  "ocamlfind"
```

Comment:
Could you either remove this dependency (I don't think jbuilder needs ocamlfind) or add the ```{build}```attribute if it is used by crowbar itself ?

Reply:
Argh, fixed this upstream but I pulled an old version here.  Thanks for pointing it out!